### ISSUE TITLE
QA Ph1_Accessibility: Missing first level heading

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#200]
There is a missing first level heading. 

Steps to reproduce: 
1. Go to https://phase1-dot-io-webapp-staging.appspot.com/io2016/
2. Using WAVE extension for Chrome add the filter WCAG 2.0 AA and view flags 
3. Note missing first level header - see screenshots for reference

Chrome 48 Mac OS X 10.11

![gio_200](https://cloud.githubusercontent.com/assets/7585604/13297455/59e83426-dae7-11e5-8a9f-46f6de57d6b7.png)
